languages/java/exercises/concept/booleans/src/test/java/QuestLogicTest.java

```diff
@@ -0,0 +1,316 @@
+import org.junit.Test;
+import org.junit.Ignore;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+public class QuestLogicTest
+{
```

Comment:
The java style rules of the track expects the open curly bracket on the same line of the name of class or method.
example
```java
public class QuestLogicTest {
    ...
    ... 

    @Test
    public void cannot_execute_fast_attack_if_knight_is_awake() {

    }

    ...
    ...
}
```

Reply:
Done